starlite/openapi/controller.py

```diff
@@ -34,6 +34,11 @@ def schema_from_request(request: Request) -> "OpenAPI":
             raise ImproperlyConfiguredException("Starlite has not been instantiated with OpenAPIConfig")
         return request.app.openapi_schema
 
+    @get(path="/", media_type=MediaType.HTML, include_in_schema=False)
+    def root(self, request: Request) -> str:
+        """Endpoint that serves root content"""
```

Comment:
this comment doesn't explain what root content is. Please explain

Reply:
Refactored it to this:

```
@get(path="/", media_type=MediaType.HTML, include_in_schema=False)
def root(self, request: Request) -> str:
    """
    This route handler serves Redoc API documentation at the root path.
    Override this handler to serve custom templates or other API documentation, such as
    `render_swagger_ui`, `render_stoplight_elements`, a template, or your own content.
    """
    return self.render_redoc(request)
```
What do you think?

Reply:
Looks good